docs/advanced_usage.md

```diff
@@ -0,0 +1,232 @@
+
+A threshold of `1` indicates that only one of the keys specified in `KEY IDS` is required to publish to this delegation. To remove a delegation role entirely, or just individual keys and/or paths, use the `notary delegation remove` command:
+
+```
+$ notary delegation remove example.com/user targets/releases
+
+Are you sure you want to remove all data for this delegation? (yes/no)
+yes
+
+Forced removal (including all keys and paths) of delegation role targets/releases to repository "example.com/user" staged for next publish.
+```
+
+You can remove individual keys and/or paths by passing keys as arguments, and/or
+paths under the `--paths` flag. Use `--all-paths` to clear all paths for this
+role. If you specify all key IDs currently in the delegation role, you will
+delete the role entirely.
+
+To add targets to a specified delegation role, we can use the `notary add`
+command with the `--roles` flag.
+
+You must have imported an appropriate delegation key for this role.  To do so,
+you can run `notary key import <KEY_FILE> --role user` with the private key PEM
+file, or drop the private key PEM in `private/tuf_keys` as `<KEY_ID>.key` with
+the `role` PEM header set to `user`.
+
+```
+$ notary add example/collections delegation/path/target delegation_file.txt --roles=targets/releases
+```
+
+In the preceding example, you add the target "delegation/path/target" to
+repository "example/collections" staged for next publish. The file
+`delegation_file.txt` is a target `delegation/path/target` using the delegation
+role `targets/releases`. This target's path is valid because it is prefixed by
+the delegation role's valid path.  
+
+The `notary list` and `notary remove` commands can also take the `--roles` flag
+to specify roles to list or remove targets from.  By default, this operates over
+the base `targets` role.
+
+To remove this target from our delegation, use the `notary remove` command with
+the same flag:
+
+```
+$ notary remove example/collections delegation/path/target --roles=targets/releases
+```
+
+## Use delegations with content trust
+
+Docker Engine 1.10 supports the usage of the `targets/releases` delegation,
+other delegation roles will be supported in a future release. By default, Engine
+1.10 with content trust enabled will attempt to retrieve images signed by the
+`targets/releases` role if it exists; if it does not exist, the Engine  will
+fall back to retrieving images signed by the default `targets` role.
+
+To use the `targets/releases` role for pushing and pulling images with content trust, follow the steps above to add and publish the delegation role with notary. When adding the delegation, the path should be restricted to the image's tag. By default, pushing with Docker Content Trust will attempt to add to the `targets/releases` role and will fallback to `targets` if the delegation does not exist.
```

Comment:
`When adding the delegation, the path should be restricted to the image's tag.`

`targets/releases` should have `--all-paths` otherwise people can't add new tags.